lore/end_users/use/README.md

```diff
@@ -73,3 +73,9 @@ what is known in git as a merge conflict.
 
 For information on how to resolve merge conflicts, refer to
 [Resolving a merge conflict on the command line](https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/).
+
+**NOTE**: Repositories which require multiple continuous integration checks
+(ex: a repository which has both BonnyCI and Travis CI) will encounter merge
+failures from BonnyCI if the other continuous integration service has checks
+that have not been met. For this reason, the use of other CI services along
+with BonnyCI is discouraged.
```

Comment:
@j2sol thanks for the clarification (this stemmed from a rec in IRC). I updated this PR accordingly.